Gems/AWSCore/Code/Platform/Windows/AWSCoreEditor_Traits_Windows.h

```diff
@@ -8,3 +8,5 @@
 #pragma once
 
 #define AWSCORE_EDITOR_RESOURCE_MAPPING_TOOL_ENABLED 1
+#define AWSCORE_EDITOR_PYTHON_DEBUG_ARGUMENT         "debug "
```

Comment:
Nit: Is the empty space after "debug" expected?

Reply:
Yes, its required, or else the string that is constructed with it will have an error